resolwe_bio/processes/amplicon/bwa_trim.yml

```diff
@@ -193,7 +193,7 @@
       echo "trimming primers"
       primerclip \
         "${NAME}_align_unsorted.sam" \
-        {{master_file.master_file.file}} \
+        {{master_file.primers.file}} \
```

Comment:
Will this work with the new masterfile?

Reply:
This is legacy process, for the old primerclip